website/static/css/docs.css

```diff
@@ -102,18 +102,17 @@
 
 hr {
   height: 1px;
-  margin-bottom: -1px;
   border: none;
   border-bottom: 1px solid $divider;
-  margin-top: 40px;
+  margin: 2em 0;
 }
 
 table {
   border: 1px solid #b0b0b0;
   border-collapse: collapse;
   display: table;
   margin: 20px 0;
-  width: 80%;
+  /* width: 80%; */
```

Comment:
Can you remove this? We try not to land commented out code. We either need it, or don't – and we can go back to it via source control.